Diagnostics/AVTester/Test-ExchAVExclusions.ps1

```diff
@@ -141,7 +141,7 @@ Write-SimpleLogfile -String "Creating EICAR Files" -name $LogFile -OutHost
 # Create the EICAR file in each path
 foreach ($Folder in $FolderList) {
 
-    [string] $FilePath = (Join-Path $Folder eicar.com)
+    [string] $FilePath = (Join-Path $Folder eicar.bat)
```

Comment:
Change to File name being a variable. Do this outside of the loop. 

$EicarFileName = "eicar.com" 


Secondly, why are we changing the file to be .bat vs .com? Most articles I have found do .com or .com.txt not .bat.

Reply:
seen a few Microsoft docs that provide that we can do this with a txt file as well. That might be even a better option?

Reply:
I agree with you, and I tested before, but the issue is that we have the txt extension in the exclusion list and if AV was correctly configured is not going to be detected:
https://docs.microsoft.com/en-us/Exchange/antispam-and-antimalware/windows-antivirus-software?view=exchserver-2019#file-name-extension-exclusions

Reply:
Is that the point? If it is correctly configured, to not get picked up?

Reply:
but in that case we can not be sure if it was for the path or for the extension and this script only test the paths

Reply:
If that is the case, then we should drop multiple file extensions to the path. Do that based off a couple of the extensions that we commonly need and then one we don't.

Reply:
I was thinking to create another function that return the extensions that we use, because it depends on the role, and we can cover all the extensions correctly.
Just to extend this script to extensions and not only paths.
But to be honest I thought it is better in another PR.
Let me know your thoughts.